packages/app/cypress/e2e/files.cy.ts

```diff
@@ -12,3 +14,18 @@ describe('App', () => {
     cy.get('[href="#/settings"]').click()
   })
 })
+
+describe('example files', () => {
```

Comment:
We already have a test for this that you can update if need be, we shouldn't duplicate the test.

Reply:
Agreed for improving the test.
We need to catch the "no file has been created" error
And we should catch the "Dialog opens ina blink and closes" error